easybuild/easyconfigs/a/AOCC/AOCC-2.3.0-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'Binary'
+
+name = 'AOCC'
+version = '2.3.0'
+
+homepage = 'https://developer.amd.com/amd-aocc/'
+description = "AMD Optimized C/C++ & Fortran compilers (AOCC) based on LLVM 11.0"
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+sources = ['aocc-compiler-%(version)s.tar']
+checksums = [
+    '9f8a1544a5268a7fb8cd21ac4bdb3f8d1571949d1de5ca48e2d3309928fc3d15',  # aocc-compiler-2.3.0.tar
+]
+
+local_gccver = '9.3.0'
+
+dependencies = [
+    ('binutils', '2.34', '', ('GCCcore', local_gccver)),
+    ('ncurses', '6.2'),
+    ('zlib', '1.2.11'),
+    ('libxml2', '2.9.10', '', ('GCCcore', local_gccver)),
+]
+
+extract_sources = True
+
+install_cmd = 'bash install.sh && mkdir -p %(installdir)s && cp -r * %(installdir)s/'
```

Comment:
There's no clean install method? 😲 

If this is the only way, we should use `cp -a` to presersve timestamps, etc.

Reply:
AOCC is a packed binary. `bash install.sh` is not installing the compiler, but creating a `setenv_AOCC.sh` that can be sourced. Maybe `bash install.sh` is not needed at all for easybuild. I have to check it again carefully. I followed closely the AOCC user guide to create a first easyconfig to have something to play around with it.

Reply:
`bash install.sh` is just creating a `lua` module and a `setenv_AOCC.sh`. It is not needed for an installation with easybuild. I removed the `install_cmd`.